### DISCUSSION TITLE
投稿 自定义主页

### DISCUSSION THREAD

**Discussion by ArgonTeam** [Meloong-Git/PCL#4558] [自定义主页]
名称：快捷主页
作者：Tu_Ruize
地址：https://turuize.top/api/custom.xaml
目前服务器在洛杉矶，之后我会移到我香港的虚拟主机上。

![屏幕截图 2024-08-20 162104](https://github.com/user-attachments/assets/958c9d0d-c880-4bf0-9135-7f7d91b7356a)
![屏幕截图 2024-08-20 162038](https://github.com/user-attachments/assets/49b1cb79-6f34-483e-8cdc-db4888882d1a)
![屏幕截图 2024-08-20 162059](https://github.com/user-attachments/assets/b5ce81f5-0391-43c6-9880-bcd2ddebb1b5)

**Comment by Deep-Dark-Forest** [comment]
感觉与简单主页没什么区别……
如果是这样的话，就大概率不会加进预设……

**Comment by bmmNbhjd** [comment]
https://github.com/Hex-Dragon/PCL2/discussions/2528#discussioncomment-10330643

**Comment by ArgonTeam** [comment]
我现在在添加必应每日一图的预览和下载功能。但是pcl在使用eventtype方法下载时没法自定义文件名。

**Comment by ArgonTeam** [comment]
成功添加了必应每日一图
![屏幕截图 2024-08-21 082417](https://github.com/user-attachments/assets/0427d821-dced-40f0-962d-85af0979f981)

**Comment by ArgonTeam** [comment]
改用了香港的虚拟主机，快了不止一点
http://tupclhome.2.996h.cn/
觉得有什么建议大家可以说

**Comment by CreeperIsASpy** [comment]
是这样的，除非你真的有信心能甩简单主页几条街，否则就别做快捷入口这类主页啦。。
因为简单主页真的做的非常好的。。而且:https://github.com/Hex-Dragon/PCL2/discussions/2528#discussioncomment-9641112

**Comment by ArgonTeam** [comment]
我们跑路了，反正也没啥用处了，，
